models/management.cattle.io.cluster.js

```diff
@@ -1,5 +1,69 @@
+import { capitalize } from 'lodash';
+import day from 'dayjs';
+import { escapeHtml } from '@/utils/string';
+import { DATE_FORMAT } from '@/store/prefs';
+
 export default {
   isReady() {
     return this.hasCondition('Ready');
-  }
+  },
+
+  configName() {
+    const allKeys = Object.keys(this.spec);
+    const configKey = allKeys.find( kee => kee.includes('Config'));
+
+    return configKey;
+  },
+
+  kubernetesVersion() {
+    const configName = this.configName;
+
+    return this.spec[configName].kubernetesVersion;
+  },
+
+  createdDisplay() {
+    const dateFormat = escapeHtml( this.$rootGetters['prefs/get'](DATE_FORMAT));
+
+    return day(this.metadata.creationTimestamp).format(`${ dateFormat }`);
+  },
+
+  displayProvider() {
+    const configName = this.configName.toLowerCase();
+    // const pools = get(this, 'nodePools');
+    // const firstPool = (pools || []).objectAt(0);
+
+    switch ( configName ) {
```

Comment:
```
const key = `cluster.provider.${configName}`;

if ( this.$store.getters['i18n/exists'](key) ) {
  return this.$store.getters['i18n/t'](key);
} else if (this.status?.driver && configName) {
  return capitalize(this.status.driver);
} else {
  return this.$store.getters['i18n/t']('cluster.provider.imported');
}
```

Reply:
(exists is in https://github.com/rancher/dashboard/pull/409/commits/543bce9a409e7b1e39c8ef3f8bc30e61105d5d05 , rebase)